lib/matplotlib/tests/test_contour.py

```diff
@@ -154,6 +154,27 @@ def test_given_colors_levels_and_extends():
         plt.colorbar(c, ax=ax)
 
 
+@image_comparison(['contour_log_locator.png'], style='mpl20', remove_text=True)
```

Comment:
Maybe do this as an svg test and keep the text?  The issue is the labels are wrong not the tick locations.